index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+
+    <link rel="stylesheet" href="styles.css" />
+    <script type="text/javascript" src="script.js"></script>
+
+</head>
+  <body>
+    <div id="form-data" class="main-container">
+      <h1 id="title">Microverse Trials with Justin and Alexander</h1>
+      <p id="description" class="description">
+        This is Alex and Justin's survey website. Please fill in this survey
+        below
+      </p>
+      <form onsubmit ="return OnSubmit();" id="survey-form" class="form">
+        <label for="name" id="name-label" class="label-input">Name</label>
+        <input id="name" name ="info" type="text" placeholder="Name" required />
+        <label for="email" id="email-label" class="label-input">Email</label>
+
+        <input id="email" name ="info" type="email" placeholder="Email" required />
+        <label for="number" id="number-label" class="label-input">Number</label>
+        <input
+          id="number"
+          value="0"
+          type="number"
+          min="0"
+          max="100"
+          placeholder="Number"
+          name ="info"
+          required
+        />
+
+        <select name="info" id="dropdown" required class="continents">
+          <option value="">--Select your continent--</option>
+          <option value="Africa">Africa</option>
+          <option value="North America">North America</option>
+          <option value="South America">South America</option>
+          <option value="Europe">Europe</option>
+          <option value="Asia">Asia</option>
+          <option value="Australia">Australia</option>
+          <option value="Antarctica">Antarctica</option>
+        </select>
+
+        <p>Select your favourite code editor:</p>
+        <div>
+          <input type="radio" id="emacs" name="fav_language" value="Emacs" class="radio-buttons"/>
+          <label for="html">Emacs</label>
+        </div>
+
+        <div>
+          <input type="radio" id="vim" name="fav_language" value="Vim" class="radio-buttons"/>
+          <label for="html">Vim</label>
+        </div>
+
+        <div>
+          <input type="radio" id="atom" name="fav_language" value="Atom" class="radio-buttons"/>
+          <label for="html">Atom</label>
+        </div>
+
+        <div>
+          <input type="radio" id="vscode" name="fav_language" value="VSCode" class="radio-buttons"/>
+          <label for="html">VSCode</label>
+        </div>
+        <div>
+          <input
+            type="radio"
+            id="sublime text"
+            name="fav_language"
+            value="sublime text"
+            class="radio-buttons"/>
+          <label for="html">Sublime</label>
+        </div>
```

Comment:
Instead of using a `<div>` tag to enclose each of your groups of code editors, it would be more proper to enclose them in a `<li>` tag, for proper use of HTML tag semantics in your codebase. 
```suggestion
     <ul>
          <li>
            <input type="radio" id="Emacs" value="Emacs" class="radio-buttons"/>
            <label for="Emacs">Emacs</label>
          </li>
          <li>
            <input type="radio" id="Vim" value="Vim" class="radio-buttons"/>
            <label for="Vim">Vim</label>
          </li>
          <li>
            <input type="radio" id="Atom" value="Atom" class="radio-buttons"/>
            <label for="Atom">Atom</label>
          </li>
          <li>
            <input type="radio" id="VSCode" value="VSCode" class="radio-buttons"/>
            <label for="VSCode">VSCode</label>
          </li>
          <li>
              <input
              id="Sublime"
              type="radio"
              value="sublime text"
              class="radio-buttons"/>
              <label for="Sublime">Sublime</label>
          </li>
        </ul>
```